hw6_protein.py

```diff
@@ -212,6 +212,16 @@ def findAminoAcidDifferences(proteinList1, proteinList2, cutoff):
 Returns: None
 '''
 def displayTextResults(commonalities, differences):
+    print('The following proteins occurred in both DNA Sequences:')
+    for o in commonalities:
+        for w in o:
+            rem=['Start','Stop']
+            if w not in rem:
+                print(w)

```

Comment:
Common proteins are not displayed as stated in lms. Please check and make required changes.

Reply:
resolved issues